Add whitelist configuration on bruteforce_configuration.rst (#11305)
nextcloud/documentation: admin_manual/configuration_server/bruteforce_configuration.rst

After a successful login the attempts will be cleared. And once a user is
properly authenticated they will no longer be hit by the delay.

Exclude IP addresses from brute force protection
------------------------

> It's possible to exlude IP addresses from the brute force protection.

Reviewer: ```suggestion
It's possible to exlude IP addresses from the brute force protection.

```

After a successful login the attempts will be cleared. And once a user is
properly authenticated they will no longer be hit by the delay.

> Brute-force IP whitelist

Reviewer: ```suggestion
Exclude IP addresses from brute force protection
```

------------------------

> You can set an whitelist of IP adress.

Reviewer: ```suggestion
It's possible to exlude IP addresses from the brute force protection.
```

> This can be set in Admin settings > Security
> You need to have the native app `bruteforcesettings` enabled.

Reviewer: ```suggestion
- Enable the bruteforcesettings app
- Login as admin and go to Administration settings -> Security
```

> Note that any whitelisted IP can perform authentication attempts without any
> throttling.  For security reasons, it is recommended to whitelist as few hosts
> as possible or ideally even none at all. 

Reviewer: ```suggestion
Note that any excluded IP address can perform authentication attempts without any throttling.  Its best to exclude as few IP addresses as you can, or even none at all. 
```
